feature-libs/user/account/components/login-form/login-form.component.ts

```diff
@@ -17,16 +23,21 @@ import { LoginFormComponentService } from './login-form-component.service';
   standalone: false,
 })
 export class LoginFormComponent {
+  @ViewChild('loginForm') loginForm: ElementRef<HTMLElementTagNameMap['form']>;
+
   constructor(protected service: LoginFormComponentService) {
     useFeatureStyles('a11yPasswordVisibliltyBtnValueOverflow');
   }
 
   form: UntypedFormGroup = this.service.form;
   isUpdating$: Observable<boolean> = this.service.isUpdating$;
+  csrf$ = this.service.csrf$;
+  action = this.service.action;
+  method = this.service.method;
 
   @HostBinding('class.user-form') style = true;
 
   onSubmit(): void {
-    this.service.login();
+    this.service.login(this.loginForm?.nativeElement);
```

Comment:
`medium` let's consider wrapping this with the feature flag so user with the feature flag disabled always call `login` method without any param

Reply:
https://github.com/SAP/spartacus/pull/20565#discussion_r2245053128

But will check it deeper, as unit tests (eg cdc login form) started failing